apps/chatbot/.env.example

```diff
@@ -21,3 +21,6 @@ CHB_ENGINE_SIMILARITY_CUTOFF=...
 CHB_ENGINE_USE_ASYNC=...
 CHB_ENGINE_USE_STREAMING=...
 CHB_QUERY_TABLE_PREFIX=chatbot-local
+GOOGLE_AND_REDIS_AWS_DEFAULT_REGION=eu-west-3
```

Comment:
I'm not sure about the naming:
We currently have a region we use for everything (eu-south-1) and a region we use only for bedrock and S3 (eu-west-3)

What do you think about changing the names to CHB_AWS_DEFAULT_REGION and CHB_AWS_BEDROCK_REGION? cc @mdciri

Reply:
same for me, I think @mdciri will update it in another pr